Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -447,16 +456,17 @@ date.
 }
 ```
 
-`begin` and `end` are in ISO8601 format.
+`begin` and `end` are in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
+format.
 
-The format is described in
-[DATE_FORMAT](../../../AQL/Functions/Date.html#dateformat).
+The formatting string format is described in
+[DATE_FORMAT()](../../../AQL/Functions/Date.html#dateformat).
 
-### Integral number
+### Integral Number
 
 This masking type replaces the value of the attribute with a random
-integral number.  It will replace the value even if it is a string,
-boolean, or false.
+integral number. It will replace the value even if it is a string,
+Boolean, or `null`.
```

Comment:
false is a Boolean, so I guess this was supposed to be string, bool, null?

Reply:
yes